README.md

```diff
@@ -155,6 +155,31 @@ func ListRepositoriesV2(ctx context.Context, client *godo.Client, registryName s
 }
 ```
 
+### Automatic Retries and Exponential Backoff
+
+The Godo client can be configured to use automatic retries and exponentional backoff for requests that fail with 429 or 500-level response codes via the [go-retryablehttp](https://github.com/hashicorp/go-retryablehttp) package. To configure the Godo client to enable usage of the go-retryablehttp, the `RetryConfig.RetryMax` must be set to enable this behavior:
+
+```
+tokenSrc := oauth2.StaticTokenSource(&oauth2.Token{
+    AccessToken: "new_token",
```

Comment:
Might be worth making this placeholder look a bit more like an API token by including the prefix. I think we do this in the public docs, but I couldn't find an example after a quick search, so not required.

```suggestion
    AccessToken: "dop_v1_xxxxxx",
```